src/power_divergence.jl

```diff
@@ -44,12 +44,14 @@ default_tail(test::PowerDivergenceTest) = :right
 pvalue(x::PowerDivergenceTest; tail=:right) = pvalue(Chisq(x.df),x.stat; tail=tail)
 
 """
-    confint(test::PowerDivergenceTest, alpha = 0.05; tail = :both, method = :sison_glaz)
+    confint(test::PowerDivergenceTest, alpha = 0.05; tail = :both, method = :auto)
 
 Compute a confidence interval with coverage 1-`alpha` for multinomial proportions using
 one of the following methods. Possible values for `method` are:
 
-  - `:sison_glaz` (default): Sison-Glaz intervals
+  - `:auto` (default): Sison-Glaz intervals unless the minimum of expected cell counts
```

Comment:
The wording here is a bit confusing. I'd say
```suggestion
  - `:auto` (default): If the minimum of the expected cell counts exceeds 100, Sison-Glaz
    intervals are used, otherwise Quesenberry-Hurst.
```